### ISSUE TITLE
Blazor Server MapFallbackToPage configuration Section on page 

### ISSUE THREAD

**Issue by gbthakkar** [dotnet/AspNetCore.Docs#23856]
Hi,

I try to follow the details in  "Blazor Server MapFallbackToPage configuration" section but...
_Host.razor name does not match with default project template file name. It is _Host.cshtml when we create a sample project in VS 2019.

and startup.cs details also does not work in my test. 

[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e07aaa4d-4ad4-778f-f74a-60d8f05d9075
* Version Independent ID: c82cf121-831d-75dd-a796-c735aad3b3e9
* Content: [Host and deploy ASP.NET Core Blazor](https://docs.microsoft.com/en-us/aspnet/core/blazor/host-and-deploy/?view=aspnetcore-5.0&tabs=visual-studio#blazor-server-mapfallbacktopage-configuration-2)
* Content Source: [aspnetcore/blazor/host-and-deploy/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/host-and-deploy/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Yes, @gbthakkar ... I think that's just a typo. I'll check in the morning and update as needed. Thanks for the report.
